### PR TITLE
Issue 62

### PR DESCRIPTION
**Pull request by eigenbeam**

### CODE REVIEW

**Review by afitzgerrell — COMMENTED** [review]
hi kevin, your additions to the readme were made to the prior readme (i.e. _not_ the one i edited yesterday). thus, maybe the most efficient route to take for your sake is to consider this 'sehr gut!! so you can release this hound. then, i'll just make the same edits to this readme that i made yesterday for the main branch readme (?)...this is assuming this readme will live on an ops-testing branch. if i'm wrong and this will live on the main branch and overwrite my pending edits, i'll just redo them there.

File: README.md [hunk 1]
```diff
@@ -31,16 +34,7 @@ or
 
     $ python3 --version
 
-Next, install [Poetry](https://python-poetry.org/) by using the [official
```
Comment (juliacollins) [inline README.md:34]: Won't we still need some of this information for anyone developing the tool? My answer is "yes" but these changes indicate you say "no." We could move some content to a `DEVELOPER.md` file if the `README` needs to be lean.

File: README.md [hunk 2]
```diff
@@ -230,14 +209,43 @@ TBD
...
+* Show the current version and the possible next versions:
+
+        $ bump-my-version show-bump
+        0.3.0 ── bump ─┬─ major ─ 1.0.0
+                       ├─ minor ─ 0.4.0
+                       ╰─ patch ─ 0.3.1
+
+* Bump the version to the desired number, for example:
+
+        $ bump-my-version bump minor
```
Comment (juliacollins) [inline README.md:234]: If I'm reading this right, you'll need to know the version you're bumping to before you update the CHANGELOG (unless there's a `bump` option to automatically replace some placeholder text with the actual version in the CHANGELOG itself). In that case, lines 222 and 223 would be more logical if moved after line 230.

File: README.md [hunk 3]
```diff
@@ -230,14 +209,43 @@ TBD
...
+
+  You will see the latest commit & tag by looking at `git log`. You
+  can then push these to GitHub to trigger the CI/CD workflow.
```
Comment (juliacollins) [inline README.md:237]: Wouldn't hurt to expressly say "you can now do `git push` to push the changes to GitHub and trigger..."

File: pyproject.toml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
-name = "nsidc-metgen"
-version = "0.1.0"
-description = "The nsidc-metgen package enables data producers as well as Operations staff managing the data ingest workflow to create metadata files conforming to NASA's Common Metadata Repository UMM-G specification."
+name = "nsidc-metgenc"
+version = "0.4.0"
```
Comment (juliacollins) [inline pyproject.toml:3]: Does `bump-my-version` automatically update this value? (Oh, I really really hope so!)

**Review by juliacollins — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (eigenbeam) [reply README.md:245]: Fixed this. Putting UNRELEASED in the CHANGELOG will get replaced by the new version.

File: README.md (hunk 3 above)
Comment (eigenbeam) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -60,27 +57,11 @@ this can be accomplished are detailed in the **AWS Credentials** section below.
 
 ## Installation of MetGenC from GitHub
```
Comment (juliacollins) [inline README.md:58]: This should be changed to "install from PyPI" text.

**Review by juliacollins — CHANGES_REQUESTED** [review]
A minor nit-pick for a heading in the README.

File: README.md (hunk 5 above)
Comment (eigenbeam) [reply README.md]: Oops, done!

**Pull request merged**